easybuild/easyconfigs/p/PyNAST/PyNAST-1.2.2-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PythonPackage'
+
+name = 'PyNAST'
+version = '1.2.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://biocore.github.io/%(namelower)s'
+description = """PyNAST is a reimplementation of the NAST sequence aligner, which has
+ become a popular tool for adding new 16s rRNA sequences to existing 16s rRNA alignments.
+ This reimplementation is more flexible, faster, and easier to install and maintain than
+ the original NAST implementation."""
```

Comment:
Same here, this is better than the values for `homepage` and `description` in the existing `PyNAST` easyconfig, do you mind changing it in the existing `PyNAST` easyconfig too?

Reply:
@boegel I added this to the existing easyconfigs.